aztec/src/test/kotlin/org/wordpress/aztec/AztecToolbarTest.kt

```diff
@@ -648,7 +648,7 @@ class AztecToolbarTest {
     @Throws(Exception::class)
     fun emptySelection() {
         editText.fromHtml("<b>bold</b><b><i>italic</i></b>")
-        editText.fromHtml("", false)
+        editText.fromHtml("")
```

Comment:
I see the previous code was using `setText` here. https://github.com/wordpress-mobile/AztecEditor-Android/pull/838/files#diff-0767c9a83c90683bf4b3661b324d2ad8L651